src/index.js

```diff
@@ -53,9 +56,26 @@ export default class EHoldings extends Component {
       showSettings,
       match: { path: rootPath }
     } = this.props;
+    let { router } = this.context;
 
     return showSettings ? (
-      <Route path={rootPath} component={SettingsRoute} />
+      <Settings
+        {...this.props}
+        pages={[
+          {
+            route: 'knowledge-base',
+            label: 'Knowledge base',
+            component: SettingsKnowledgeBaseRoute
+          },
+          {
+            route: 'root-proxy',
+            label: 'Root proxy',
+            component: SettingsRootProxyRoute
+          }
+        ]}
+        paneTitle="eHoldings"
+        activeLink={router.route.location.pathname}
+      />
```

Comment:
How do we feel about this pattern?

Reply:
ehhhhhh, so `Settings` contains a router and handles its own routes? I feel like scattering routes in many places throughout a SPA is a recipe for disaster.

If we can't prevent the bad pattern from entering our app (or contribute upstream a better fix) I think we should document that this is not a pattern that should be replicated in elsewhere in our app. ☣

Reply:
I'd much rather pass actual `<Route>`s as children to this component.

```
<Route ... component={Settings}>
  <Route name="Knowledge Base" ... component={SettingsKnowledgeBaseRoute}/>
</Route>
```

Settings component can look down at props to get the name.

```
let names = React.Children.map(this.props.children, child => child.props.name)
```

Reply:
I'm hesitant to make any more changes to `stripes-components`' `Settings` without simultaneously adding test coverage. How about the commit I just added as a compromise? It at least hides away the gook inside the `SettingsRoute`.